ci/audit.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
It looks like the permissions are wrong on this file:

```
chmod 755 ci/scripts/audit.sh
```
Should sort things out.